desc/objectives/_coils.py

```diff
@@ -1110,11 +1110,17 @@ class QuadraticFlux(_Objective):
     (B_coil + B_plasma)*n. The equilibrium is kept fixed while the
     field is unfixed.
 
+    Can also be used to find quadratic-flux-minimizing surfaces, in which
+    case a `FourierRZToroidalSurface` should be passed instead of an Equilibrium,
+    and `qfm_surface=True` should be passed into the objective.
+
     Parameters
     ----------
-    eq : Equilibrium
-        Equilibrium upon whose surface the normal field error will be minimized.
-        The equilibrium is kept fixed during the optimization with this objective.
+    eq : Equilibrium or FourierRZToroidalSurface
+        Equilibrium (or QFM surface) upon whose surface the normal field error
```

Comment:
should maybe add a note to use BoundaryError or VacuumBoundaryError for free boundary / single stage optimization